lib/decimal.ex

```diff
@@ -49,6 +49,29 @@ defmodule Decimal do
 
   import Kernel, except: [abs: 1, div: 2, max: 2, min: 2, rem: 1, round: 1]
 
+  @typedoc """
+  The coefficient of the power of `10`.  Non-negative because the sign is stored separately in `sign`.
+
+  * `non_neg_integer` - a real number
```

Comment:
Using "real" can be confusing since real numbers are floats but this is an integer.